runtime/near-vm-runner/src/cache.rs

```diff
@@ -35,3 +43,81 @@ pub(crate) fn compile_module(
     let prepared_code = prepare::prepare_contract(code, config)?;
     wasmer_runtime::compile(&prepared_code).map_err(|err| err.into_vm_error())
 }
+
+fn get_cached_name(code_hash: &Vec<u8>) -> String {
+    // For a prototype we store compiled contracts in file, later on we'll use database.
+    let dir = match env::var_os("HOME") {
+        Some(val) => format!("{}/.near/cache", val.into_string().unwrap()),
```

Comment:
Do we need to support windows?  If so does this work?  Or do we need to join the paths.

Reply:
Plan was to use RocksDB in real solution, this is just a WIP to show the concepts.